src/test/java/TrainingEvaluatorTest.java

```diff
@@ -0,0 +1,60 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class TrainingEvaluatorTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"0:1", "25:1", "30:2", "45:2", "60:2", "65:3" }, delimiter = ':')
```

Comment:
Przydadzą się testy dla warunków brzegowych.

Reply:
Poproszę o jak podpowiedz, nie wiem jak przypisa null albo "" do inta?

Reply:
Mimo, że logicznie może może być to niemożliwe, to warto przetestować np. liczby ujemne